### ISSUE TITLE
Bug: While importing the annotation class ID is wrong

### ISSUE THREAD

**Issue by waqarsqureshi** [open-mmlab/labelbee-client#30]
When we use import to mask. The mask image is saved with the wrong IDs it starts from 1,2,3,4, irrespective the value set by the person in the properties

0 is set to no attribute by default.

**Comment by Kerwin-L** [comment]
Sorry do you mean the `trainIDs`?
`xxx-color-list.json` is  a mapping of attributes to color and grayscale value, in which the `trainIDs` is the grayscale value. Grayscale value starts from 1, and the value 0 means the background

**Comment by waqarsqureshi** [comment]
Another issue is the TrainIDs are saved as color images i.e. and not as a 8bit unsign Int image.

**Comment by waqarsqureshi** [comment]
There is an option of Attribute, where you can select the key and value of your choice. I selected road=0 and background as 1. However it still used its own value i.e. 0 to background and then so on.

**Comment by waqarsqureshi** [comment]
![image](https://user-images.githubusercontent.com/9814217/168246148-1c6698f2-ff26-4658-b9c8-d21c002c598a.png)
I attended to use labels as above. However, in the jason while it is as below:
`[
  {
    "attribute": "2",
    "color": "rgb(128,0,0)",
    "colorList": [
      128,
      0,
      0,
      255
    ],
    "trainIds": 1
  },
  {
    "attribute": "0",
    "color": "rgb(0,128,0)",
    "colorList": [
      0,
      128,
      0,
      255
    ],
    "trainIds": 2
  },
  {
    "attribute": "3",
    "color": "rgb(128,128,0)",
    "colorList": [
      128,
      128,
      0,
      255
    ],
    "trainIds": 3
  },
  {
    "attribute": "1",
    "color": "rgb(0,0,128)",
    "colorList": [
      0,
      0,
      128,
      255
    ],
    "trainIds": 4
  }
]`

**Comment by Kerwin-L** [comment]
> Another issue is the TrainIDs are saved as color images i.e. and not as a 8bit unsign Int image.

Yes, js can't export 8bit unsign Int image, so I provided the python script to solve this problem 
<img width="1041" alt="image" src="https://user-images.githubusercontent.com/21312795/168246844-20063d11-98ca-43fd-bc36-55ad6f1d1893.png">

**Comment by Kerwin-L** [comment]
> ![image](https://user-images.githubusercontent.com/9814217/168246148-1c6698f2-ff26-4658-b9c8-d21c002c598a.png) I attended to use labels as above. However, in the jason while it is as below: `[ { "attribute": "2", "color": "rgb(128,0,0)", "colorList": [ 128, 0, 0, 255 ], "trainIds": 1 }, { "attribute": "0", "color": "rgb(0,128,0)", "colorList": [ 0, 128, 0, 255 ], "trainIds": 2 }, { "attribute": "3", "color": "rgb(128,128,0)", "colorList": [ 128, 128, 0, 255 ], "trainIds": 3 }, { "attribute": "1", "color": "rgb(0,0,128)", "colorList": [ 0, 0, 128, 255 ], "trainIds": 4 } ]` It did not follow either of the sequence given in the attribute but rather made its own TrainIds.

The sequence of the json follows  the sequence of results. This approach does make the configuration random, i will update it later, thanks for comments.

You can now determine the current tranid by the attribute. For example, You now want `road`, whose value you configured as `0`, to get through the configuration file `[  { "attribute": "0", "color": "rgb(0,128,0)", "colorList": [ 0, 128, 0, 255 ], "trainIds": 2 } ]`, its trainIDs (grayscale) is 2.
